client/src/pages/components/dumbComponents.js

```diff
@@ -0,0 +1,144 @@
+        </div>
+    )
+}
+
+export function getName(onChangeCallback, onClickCallback) {
+    return (
+        <div>
+            <h2>Please Enter your name</h2>  
+            <div className="col-md-4 mb-3">
+                <div className="input-group">
+                    <input type="text" 
+                        className="form-control" 
+                        placeholder="Name"
+                        aria-label="Enter name for email greeting" 
+                        onChange={onChangeCallback}
+                        required/>
+                </div>
+            </div>
+            <button className="btn btn-primary" 
+                type="submit" 
+                onClick={onClickCallback}>Submit</button>
+        </div>
+    )
+}
+
+export function getSubject(onChangeCallback, onClickCallback) {
+    return (
+        <div>
+            <h2>Please Enter a Subject</h2>  
+            <div className="col-md-4 mb-3">
+                <div className="input-group">
+                    <input type="text" 
+                        className="form-control" 
+                        placeholder="Subject" 
+                        aria-label="Enter subject for email" 
+                        onChange={onChangeCallback}
+                        required/>
+                </div>
+            </div>
+            <button className="btn btn-primary" type="submit" onClick={onClickCallback}>Submit</button>
+        </div>
+    )
+}
+
+export function getEmail(onChangeCallback, onClickCallback) {
+    return (
+        <div>
+            <h2>Please Enter Your Email</h2>  
+            <div className="col-md-4 mb-3">
+                <div className="input-group">
+                    <div className="input-group-prepend">
+                    <span className="input-group-text" id="inputGroupPrepend">@</span>
+                    </div>
+                    <input type="text" 
+                        className="form-control" 
+                        id="validationCustomUsername" 
+                        placeholder="Email"
+                        aria-label="Enter email for message sending"  
+                        aria-describedby="inputGroupPrepend" 
+                        onChange={onChangeCallback}
+                        required/>
+                </div>
+            </div>
+            <button className="btn btn-primary" type="submit" onClick={onClickCallback}>Submit</button>
+        </div>
+    )
+}
+
+export function stateRecipientSelection(onChangeCallback, onClickCallback) {
+    return (
+        <div>
+            <h2>What state officials would you like send emails to?</h2>
+            <div className="form-group">
+                <label htmlFor="state" className="col-sm-2 control-label">State</label>
+                <div className="col-sm-5">
+                    <select className="form-control" id="state" name="state" onBlue={onChangeCallback}>
```

Comment:
This select would be more efficient as array of state abbreviations to state names. Then you can do something along the lines of:

```js 
// states.json
export const states = [ { abbrev: "ALL", name: "Send To All"}, {abbrev: "AK", name: "Alaska"}, ...];
```

```js
import { states } from './states';
. . .
<select className="form-control" id="state" name="state" onBlue={onChangeCallback}>
    { states.map(state => (<option value={state.abbrev} key={state.abbrev}>{state.name}</option>) }
</select>
```